### DISCUSSION TITLE
Helpp

### DISCUSSION THREAD

**Discussion by secretchowc** [BeansGalaxy/Beans-Backpacks-2#171] [Feedback]
Any way to remove this? I don't like it at all, I would like some advice on how to change this, as the second image shows the comfort I would like. 
![Captura de pantalla 2024-06-23 181827](https://github.com/BeansGalaxy/Beans-Backpacks-2/assets/86332442/01976ffc-d973-4e2b-a799-11361e04b3c8)

![image](https://github.com/BeansGalaxy/Beans-Backpacks-2/assets/86332442/5d45551e-9e65-455f-8a62-b1f0a0228fc9)

**Comment by BeansGalaxy** [comment]
You can move the slot somewhere else by using the config. Install Mod Menu if you're on fabric. 
1. click mods on the main menu
2. Find Beans Backpacks and press the configure button
3. At the bottom of "Common Config" edit `back_slot_pos`
4. Move it to where you'd like

If you want to completely hide it open your installation and the config folder, inside `beans-backpacks-common` change `back_slot_pos` to something like `[10000, 10000]` and it'll move it off screen.

Joining a server will put it to where the server's config has it for networking reasons but this'll work in singleplayer or your servers
